src/templater.sh

```diff
@@ -48,6 +48,7 @@ function templater() {
 
   # cat "${templateDst}"
 
+  # shellcheck disable=SC2016
   if grep '${' "${templateDst}" && [ "${ALLOW_REMAINDERS}" = "0" ]; then
```

Comment:
(you'll first need to accept the invitation before new permissions are activated btw)